consent-ui/src/app/ais/entry-page/initiation/accounts/accounts-consent-review/accounts-consent-review.component.html

```diff
@@ -36,14 +36,40 @@ <h2>Access permission for accounts in {{ aspspName }}</h2>
         <div class="flex-container">
           <img src="assets/icons/icons8-approval.png" alt="approval" class="icons" />
           <p>
-            The consent will be valid until <strong>{{ aisConsent.consent.validUntil }}</strong>
+            The consent will be valid until:
```

Comment:
Adjust CSS, so that controls are not that large, looks a bit awkward
![image](https://user-images.githubusercontent.com/18463491/118772282-3f306300-b88c-11eb-89dd-b4a7b7493ae7.png)

Reply:
fixed fields size